recipe/build.sh

```diff
@@ -9,15 +9,19 @@ then
 fi
 export EXTRA_CMAKE_ARGS
 
-mkdir "build_${CMAKE_CONFIG}"
+mkdir -p "build_${CMAKE_CONFIG}"
 pushd "build_${CMAKE_CONFIG}"
+
 cmake -G "Ninja" \
-    -DCMAKE_BUILD_TYPE:STRING="${CMAKE_CONFIG}" \
-    -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
-    -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON \
-    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
+    -DBUILD_SHARED_LIBS=ON \
+    -DCMAKE_INSTALL_LIBDIR=lib \
+    ${CMAKE_ARGS} \
```

Comment:
Where does this come from?

Reply:
from the CMAKE feedstock afaik. isuruf introduced this afaik.

Reply:
And this exported somehow? Is conda-build passing this through somewhere? Or is it in an activation script?